lib/utils/node-module-template.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
+define('APP_NAME/node-asset-manifest', function() {
```

Comment:
So I did that originally, but ESLint whines that it can't parse the file (even though I've set it to disabled)